packages/run-protocol/README.md

```diff
@@ -25,3 +25,99 @@ Each interest charging period (say daily) the actual debts in all vaults are aff
 - a debtSnapshot on the vault by which one can calculate the actual debt
 
 To maintain that the keys of vaults to liquidate are stable requires that its keys are also time-independent so they're recorded as a "normalized collateralization ratio", with the actual collateral divided by the normalized debt.
+
+## Reading data off-chain
+
+VaultFactory publishes data using StoredPublishKit which tees writes to off-chain storage. These can then be followed off-chain like so,
+```js
+  const key = `published.vaultFactory.metrics`; // or whatever the stream of interest is
+  const leader = makeDefaultLeader();
+  const follower = makeFollower(storeKey, leader);
+  for await (const { value } of iterateLatest(follower)) {
+    console.log(`here's a value`, value);
+  }
+```
+
+The canonical keys (under `published`) are as follows. Non-terminal nodes could have data but don't yet. A `0` indicates the index of that child in added order. To get the actual key look it up in parent. High cardinality types get a parent key for enumeration (e.g. `vaults`.)
+- `published`
+    - `vaultFactory`
+        - `governance`
+        - `metrics`
+        - `manager0`
+            - `metrics`
+            - `governance`
+            - `vaults`
+              - `vault0`
+    - `amm`
+        - `metrics`
+        - `governance`
+        - `pool0`
+            - `metrics`
+    - `psm`
+        - `governance`
+        - ~~`metrics`~~ TODO
+    - `stakeFactory`
```

Comment:
FWIW, metrics and individual BLD boost vaults are TODO too: https://github.com/agoric/agoric-sdk/issues/4653
(not critical / important to add them here now)